src/Testable/Internal.elm

```diff
@@ -2,12 +2,14 @@ module Testable.Internal exposing (..)
 
 import Http
 import Time exposing (Time)
+import Platform.Cmd
 
 
 type Cmd msg
     = None
     | TaskCmd (Task msg msg)
     | Batch (List (Cmd msg))
+    | PortCmd (Platform.Cmd.Cmd msg)
```

Comment:
I'm not a huge fan of this, but since ports are enforced at a language level to have `Platform.Cmd.Cmd` in their signature I don't think there's any way around it either.

Since `PortCmd` is just wrapping `Platform.Cmd.Cmd` it is essentially just a `WrappedCmd` (or something), not necessarily even confined to ports. Any generic `Cmd` could be wrapped in this.

This still does have some use though since it would mean this package could be used by any application with any `Cmd` type, whether it be a port, or [`Navigation`](http://package.elm-lang.org/packages/elm-lang/navigation/1.0.0/Navigation#newUrl), or [`Websocket`](http://package.elm-lang.org/packages/elm-lang/websocket/1.0.1/WebSocket#send) or any of the various "web API" packages that are supposed to be coming down the line in the coming months.

There could be new types added to `Testable.Internal.Cmd` for each of these and future types (mainly for making testing assertions/failure messages cleaner) but having a general wrapped type would mean that this package wouldn't have to stay "ahead" of every single other effects manager package in order to be usable. You would never be in the situation of "I want to use this package but can't because it hasn't added support for ___ yet". You could just use the wrapped type.

Even if you couldn't cleanly assert that that specific `Cmd` was called the way you wanted it to be, you wouldn't wind up not being able to test your HTTP requests because elm-testable hadn't implemented ports or Websocket.send or some other `Cmd`.